lib/test.js

```diff
@@ -70,6 +70,11 @@ Object.keys(assert).forEach(function (el) {
 	};
 });
 
+// Workaround for power-assert
```

Comment:
Unfortunately, this workaround is necessary for now (`_capt` and `_expr` methods are needed at the top level).

To make them look less messy, I'll introduce small helper method like
```js
empower.makeCapturable(Test.prototype);
```
(looking for a better name though)